### ISSUE TITLE
How to display 2 lines of label text?

### ISSUE THREAD

**Issue by whruiqlik** [AnyChart/AnyChart-Qlik#30]
Hi, How to display 2 lines of label text? Thank you.

Token currently in use：{%Value}{decimalsCount:0,groupsSeparator:\,} ({%PercentValue}{decimalsCount:1,zeroFillDecimals:true}% )

![20191207112307](https://user-images.githubusercontent.com/43289168/70368442-718f1000-18e5-11ea-91b8-4bd6bf216855.png)

**Comment by Shestac92** [comment]
@whruiqlik 
To add a line break you can add `\n` chars. 
For example, you can modify your string token like this: `{%Value}{decimalsCount:0,groupsSeparator:,}/n({%PercentValue}{decimalsCount:1,zeroFillDecimals:true}% )`

**Comment by whruiqlik** [comment]
I tried to add "/ N", but still couldn't display two lines.

![20191209163826](https://user-images.githubusercontent.com/43289168/70420497-d2177c00-1aa2-11ea-8fbd-b0bb027ecc9f.png)

**Comment by whruiqlik** [comment]
{%Value}{decimalsCount:0,groupsSeparator:,} /n {%PercentValue}{decimalsCount:1,zeroFillDecimals:true}%

**Comment by Shestac92** [comment]
@whruiqlik 
You are using `/n`, but the special code is `\n` - with a reverse slash.

**Comment by whruiqlik** [comment]
I tried to use " \n ", but the label could not display 2 lines. Is there any other reason for this problem?

![20191210121710](https://user-images.githubusercontent.com/43289168/70495029-55d77400-1b48-11ea-8d31-667cb7a9cb3c.png)

**Comment by Shestac92** [comment]
@whruiqlik 
I've just tried in the latest version of the extension and it works properly.
![Screenshot 2019-12-10 at 15 40 22](https://user-images.githubusercontent.com/24609062/70505729-ac05e080-1b63-11ea-9fb4-c14abf4bf64c.png)

I'm afraid that you are using not the latest version. Please, can you check the version of the extension in the `About` tab, like below:
![Screenshot 2019-12-10 at 15 40 41](https://user-images.githubusercontent.com/24609062/70505764-c5a72800-1b63-11ea-915b-3b1ff59bad03.png)

**Comment by whruiqlik** [comment]
The anychart I used is an old version. Is the latest version of anychart compatible with June 2018 version of Qlik Sense? Thank you.

![20191211164650](https://user-images.githubusercontent.com/43289168/70613958-8dc3e180-1c44-11ea-9166-1a168ce15b6f.png)

**Comment by Shestac92** [comment]
@whruiqlik 
Yes, the latest version of AnyChart extensions are compatible with Qlik Sense 2018

**Comment by whruiqlik** [comment]
Thank you very much.

**Comment by whruiqlik** [comment]
Where can I download the latest full version of AnyChatr? Thank you.

**Comment by Shestac92** [comment]
@whruiqlik 
If you have the original link in your mailbox for downloading the extensions you can use it to get the latest version. Otherwise, fill the form at [the download page](https://qlik.anychart.com/download/) and you will receive the download link.

**Comment by whruiqlik** [comment]
Ok, thanks!
